File: python3/FizzBuzz_v1.py
#!/usr/bin/env python3
# python3でのFizzBuzz

def main():
        for i in range(100):
            if i % 3 == 0 and i % 5 == 0:
                print(i,"FizzBuzz.")
            elif i % 3 == 0:
                print(i,"Fizz.")
            elif i % 5 == 0:
                print(i,"Buzz.")
            else:
                print(i)
        

if __name__ == '__main__':
    main()

File: Ruby/FizzBuzz_v1.rb
#! /usr/bin/env ruby
# RubyでFizzBuzz.

def FizzBuzz
    for i in 0..100 do
        if i % 5 == 0 && i % 3 == 0
            puts "#{i} FizzBuzz."
        elsif i % 3 == 0 
            puts "#{i} Fizz."
        elsif i % 5 == 0
            puts "#{i} Buzz."
        else
            puts "#{i}"
        end
    end
end

FizzBuzz()
